tcf_website/static/course/javascript/course_data.js

```diff
@@ -28,12 +33,13 @@ const loadData = data => {
     exist(data.average_amount_writing) ? document.getElementsByClassName("writing-bar")[0].style.width = 100 * data.average_amount_writing / data.average_hours_per_week + "%" : null;
     exist(data.average_amount_group) ? document.getElementsByClassName("group-bar")[0].style.width = 100 * data.average_amount_group / data.average_hours_per_week + "%" : null;
     exist(data.average_amount_homework) ? document.getElementsByClassName("homework-bar")[0].style.width = 100 * data.average_amount_homework / data.average_hours_per_week + "%" : null;
+    /* eslint-enable no-unused-expressions */
 };
 
-const createChart = grades_data => {
-    const chart_data = {
+const createChart = gradesData => {
+    const chartData = {
```

Comment:
I renamed `chart_data` and `grades_data` to use eslint's preferred camelCase styling.